controllers/csm_controller_test.go

```diff
@@ -364,7 +365,7 @@ func (suite *CSMControllerTestSuite) TestCsmUpgradePathInvalid() {
 		annotations = make(map[string]string)
 	}
 	if _, ok := annotations[configVersionKey]; !ok {
-		annotations[configVersionKey] = jumpUpgradeConfigVersion
+		annotations[configVersionKey] = invalidConfigVersion
```

Comment:
we can revert this change because the unit tests are fixed now

Reply:
reverting that change should fix the coverage as well, I think?

Reply:
the jumpUpgradeConfigVersion is being used in 2 different tests right now https://github.com/dell/csm-operator/blob/dc1358b4bc3997ca93bf5a9abaee104752419bc0/controllers/csm_controller_test.go#L341